authenticate/forms.py

```diff
@@ -0,0 +1,26 @@
+from django import forms
+
+
+# Registration Form
+class RegisterForm(forms.Form):
+  first_name = forms.CharField(max_length=50, required=True)
+  last_name = forms.CharField(max_length=50, required=True)
+  username = forms.CharField(max_length=50, required=True)
+  email = forms.EmailField(required=True)
+  password1 = forms.CharField(widget=forms.PasswordInput(), required=True)
+  password2 = forms.CharField(widget=forms.PasswordInput(), required=True)
+
+
+# Login Form
+class LoginForm(forms.Form):
+  username = forms.CharField(max_length=50, required=True)
+  password = forms.CharField(widget=forms.PasswordInput(), required=True)
+
+
+# Profile Form
+class ProfileForm(forms.Form):
+  first_name = forms.CharField(max_length=50, required=True)
+  last_name = forms.CharField(max_length=50, required=True)
+  email = forms.EmailField(required=True)
+  password1 = forms.CharField(widget=forms.PasswordInput(), required=False)
+  password2 = forms.CharField(widget=forms.PasswordInput(), required=False)
```

Comment:
nit: please make sure to have a new line at the end of each file (Pycharm and other IDE's have the feature to put a new line at the end of the file automatically if not already present)

Reply:
Added